packages/fcl-react-native/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@onflow/fcl-react-native",
+  "version": "1.8.1",
+  "description": "Flow Client Library",
+  "license": "Apache-2.0",
+  "author": "Dapper Labs <dev@dapperlabs.com>",
+  "homepage": "https://onflow.org",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/onflow/flow-js-sdk.git"
+  },
+  "bugs": {
+    "url": "https://github.com/onflow/flow-js-sdk/issues"
+  },
+  "jest": {
+    "testEnvironment": "jsdom",
+    "globals": {
+      "PACKAGE_CURRENT_VERSION": "TESTVERSION"
+    }
+  },
+  "devDependencies": {
+    "@onflow/fcl-bundle": "^1.4.1",
+    "@onflow/typedefs": "^1.2.1",
+    "@types/estree": "^1.0.1",
+    "@types/node": "^18.13.0",
+    "eslint": "^8.35.0",
+    "eslint-plugin-jsdoc": "^46.9.0",
+    "jest": "^29.5.0",
+    "jest-environment-jsdom": "^29.5.0",
+    "typescript": "^4.9.5"
+  },
+  "source": "src/fcl-react-native.ts",
+  "main": "dist/fcl-react-native.js",
+  "module": "dist/fcl-react-native.module.js",
+  "unpkg": "dist/fcl-react-native.umd.min.js",
+  "types": "types/fcl-react-native.d.ts",
+  "scripts": {
+    "pain": "npm publish --tag pain",
+    "alpha": "npm publish --tag alpha",
+    "prepublishOnly": "npm test && npm run build",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "build": "npm run lint && fcl-bundle",
+    "build:types": "tsc",
+    "start": "fcl-bundle --watch",
+    "lint": "eslint ."
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.18.6",
+    "@onflow/fcl-core": "^1.8.1",
```

Comment:
imo we should lock this dependency (i.e. `"1.8.1"`).  I have the feeling we're going to have a difficult time following semver with this package since this is was previously functionality internal to FCL (@onflow/fcl-react-native will probably end up tightly coupled/reliant on an exact version of @onflow/fcl-core).

Also if it's not locked we risk users encountering different "FCL" behaviour in the same version of FCL.  This is honestly a problem with all of our packages and I wonder if we should enforce this requirement across the board (but this is a different discussion maybe).